### ISSUE TITLE
cmake fails in opencl.cmake (Trac #1108)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1102]
Running cmake on OSX fails with the following error

CMake Error at cmake/tools/opencl.cmake:51 (REPORT_FIND):
  report_find Macro invoked with incorrect arguments for macro named:
  report_find
Call Stack (most recent call first):
  cmake/tools.cmake:72 (include)
  CMakeLists.txt:82 (include)

When I revert this commit it finishes succesfully: 

http://code.icecube.wisc.edu/projects/icecube/changeset/2193/IceTray/projects/cmake/trunk/tools/opencl.cmake

Migrated from https://code.icecube.wisc.edu/ticket/1108
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:57:21", 
    "description": "Running cmake on OSX fails with the following error\n\nCMake Error at cmake/tools/opencl.cmake:51 (REPORT_FIND):\n  report_find Macro invoked with incorrect arguments for macro named:\n  report_find\nCall Stack (most recent call first):\n  cmake/tools.cmake:72 (include)\n  CMakeLists.txt:82 (include)\n\nWhen I revert this commit it finishes succesfully: \n\nhttp://code.icecube.wisc.edu/projects/icecube/changeset/2193/IceTray/projects/cmake/trunk/tools/opencl.cmake", 
    "reporter": "lraedel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251041579296", 
    "component": "cmake", 
    "summary": "cmake fails in opencl.cmake", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-11T20:17:16", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150811T20:17:39: nega changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150811T20:28:46: nega commented:

> worksforme on 10.9.5
> 
> i'll have to check later on 10.10, but it shouldn't make a difference.

**Comment by icecube-trac** [comment]
Trac update at 20150812T17:37:50: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "accepted" to "closed"
* jvansanten commented:

> In changeset 2202:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2202"
> Unset OPENCL_LIBRARIES to force it to evaluate to False with CMake 3.2.1. Fixes #1102.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:57:21: nwhitehorn commented:

> In changeset 170077:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170077"
> Unset OPENCL_LIBRARIES to force it to evaluate to False with CMake 3.2.1. Fixes #1102.
> ```
